.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Ensure STC is installed with Pyccel'
+
+runs:
+  using: "composite"
+  steps:
+    - name: check stc installation
+      shell: python
+      run: |
+        import pyccel
+        from os.path import exists, join
+        pyccel_loc = pyccel.__path__[0]
+        print(pyccel_loc)
+        assert(exists(join(pyccel_loc,'extensions/STC/include/stc')))
+        assert(not exists(join(pyccel_loc,'extensions/STC/docs')))
```

Comment:
Don't you need an if condition here so it doesn't complain in editable mode?

Reply:
Answer, yes:
https://github.com/pyccel/pyccel/actions/runs/8468392946/job/23201312501

Reply:
you're right, I will add it in my next commit.

Reply:
I added a condition her :  787aeb9. Let me know if it's the right way to solve this problem.

Reply:
I'm not sure this is the right way to do this. A user installation (`--user`) is writable but is not necessarily editable. I think the simplest is to parametrise the test

Reply:
The link tells you what to put in the action.yml file. Maybe you can look at our `post_completed` action for an example